src/transformers/models/graphormer/fairseq_utils.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This snippet of code comes from fairseq
+# https://github.com/facebookresearch/fairseq/blob/dd0079bde7f678b0cd0715cbd0ae68d661b7226d/fairseq/modules/quant_noise.py
+# https://github.com/facebookresearch/fairseq/blob/dd0079bde7f678b0cd0715cbd0ae68d661b7226d/fairseq/modules/layer_drop.py
+# It is licensed under the MIT license.
+
+import torch
+import torch.nn as nn
+
+
+def quant_noise(module, p, block_size):
```

Comment:
Not convinced we need a separate module for just this function and module. Those two can just go in `modeling_graphormer.py` IMO.